stylus/components/button.styl

```diff
@@ -348,3 +350,109 @@ $button--full
 
 $button--narrow
     min-width auto
+
+/*------------------------------------*\
+  Action button
+\*------------------------------------*/
+$actionbtn
+    @extend $button
+    border-color silver
+    text-transform none
+    max-width 12.5rem
+    min-height 2rem
+    width 100%
+    padding-right .5rem
+    text-align left
+
+    > span
+        justify-content flex-start
```

Comment:
For my culture, why is it necessary to add ?

Reply:
because I `@extend $button` which overwrite the default value